src/client.js

```diff
@@ -9,14 +9,16 @@ const cache = require('./cache');
 module.exports = {
     generate_token,
     refresh_token,
-    revoke_token
+    revoke_token,
+    remove_cache_only,
 };
 
 function generate_token (scopes) {
     const scopes_string = _(scopes).sort().sortedUniq().join(' ');
     const cached_token_result = cache.get('client', scopes_string);
 
     if (cached_token_result) {
+        console.log('CACHED TOKEN', cached_token_result);
```

Comment:
console? we don't have logger configured? If not, let's don't log anything

Reply:
Aw. Sorry. I forgot to removed it. :sweat_smile: I'll update this one.